js/utils/timestamp.js

```diff
@@ -106,6 +112,7 @@ o2.Utilities.phpToMoment = function( s ) {
 			case 'u': // Microseconds
 			case 'I': // Daylight savings time
 			case 'Z': // Timezone offset in seconds
+			case '\\': // Scape character
```

Comment:
Might there be a case for looking forward at this point and incrementing `i`? You'd have to be clever with the setting of `lookBehind` below, but it would keep the conditional logic all in the `switch` statement. 

It's more something to think to check we're taking the right approach, than anything else.